exp/README.md

```diff
@@ -0,0 +1,59 @@
+## 📋 Prerequisites
+
+- Python 3.8+
+- OpenAI API key
+- Internet connection for research APIs
+
+## 🔧 Installation
+
+1. **Install dependencies**
+   ```bash
+   pip install -r requirements.txt
+   ```
+
+2. **Set up OpenAI API Key** (Optional - can be done at runtime)
+   ```bash
+   export OPENAI_API_KEY="your-openai-api-key-here"
+   ```
+
+## 🎯 Quick Start
+
+### Method 1: Interactive Mode
+
+```bash
+python main.py
+```
+
+The script will:
+
+1. Prompt for OpenAI API key (if not in environment)
+2. Ask for target directory for solution files
+3. Start solving the default MNIST problem
+
+### Method 2: Command Line with Directory
+
+```bash
+python main.py /path/to/solution/directory
```

Comment:
@YuanmingLeee That's what we want.

We don't need a new `mle bench` abstraction, but just need this to process `lite` directories